tests/test_types.py

```diff
@@ -1341,18 +1342,130 @@ def test_int_enum_successful_for_str_int(cooking_model):
     assert repr(m.tool) == '<ToolEnum.wrench: 2>'
 
 
+def test_plain_enum_validate():
+    class MyEnum(Enum):
+        a = 1
+
+    class Model(BaseModel):
+        x: MyEnum
+
+    m = Model(x=MyEnum.a)
+    assert m.x == MyEnum.a
+
+
+def test_plain_enum_validate_json():
+    class MyEnum(Enum):
+        a = 1
+
+    class Model(BaseModel):
+        x: MyEnum
+
+    m = Model.model_validate_json('{"x":1}')
+    assert m.x == MyEnum.a
```

Comment:
```suggestion
    m = Model.model_validate_json('{"x":1}')
    assert m.x is MyEnum.a
```

I'm suggesting using `is` because `MyEnum.a == 1` is True (I think).

Reply:
== fails for plain enums like in this case, but I agree we should do the `is` since we can.